### PR TITLE
Ltc4332 spi extender

### PR DESCRIPTION
**Pull request by buha**
This is the same as #1490 but with the astyle fix.

### CODE REVIEW

**Comment by buha** [comment]
V2:
- changed the git author to myself since @Alcharyx can't sign the CLA agreement.

File: drivers/io-expander/ltc4332/ltc4332.c [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	temp_ops = desc->platform_ops;
+	desc->platform_ops = desc->parent->platform_ops;
+
+	buff = calloc(bytes_number + 1, sizeof *buff);
+	if (!buff)
+		return -ENOMEM;
+
+	memcpy(buff, data, bytes_number);
+	ret = no_os_spi_write_and_read(desc, buff, bytes_number + 1);
+
```
Comment (amiclaus) [inline drivers/io-expander/ltc4332/ltc4332.c:130]: nit: extra line

File: drivers/io-expander/ltc4332/ltc4332.c [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+	int32_t ret;
+	struct no_os_spi_init_param temp_param;
+
+	if (!param->parent)
+		return -EINVAL;
+
+	temp_param = *param;
+	temp_param.platform_ops = param->parent->platform_ops;
+	ret = no_os_spi_init(desc, &temp_param);
+
```
Comment (amiclaus) [inline drivers/io-expander/ltc4332/ltc4332.c:80]: nit: extra line

**Review by amiclaus — APPROVED** [review]

**Comment by buha** [comment]
V3:
- removed whitespaces

**Pull request merged**